Classes/EventStore/EventTransport.php

```diff
@@ -0,0 +1,58 @@
+<?php
+namespace Neos\Cqrs\EventStore;
+
+/*
+ * This file is part of the Neos.EventStore package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Cqrs\Event\EventInterface;
+
+/**
+ * Wrapper for a event that was loaded from the Event Store.
+ * It contains the raw event data including version & metadata (storedEvent)
+ * and the converted EventInterface instance (event)
+ */
+final class EventTransport
```

Comment:
Why do we need a wrapper for storedEvent + domain event? The latter is deterministically extracted/deserialized from the first and should only happen once on-the-fly when loading from store.